pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
+
+stdenv.mkDerivation rec {
+ baseurl = "https://redwax.eu/dist/rs/";
```

Comment:
We don't typically share variables like this across different packages. It might seem tedious for this PR, but it's just a few lines to copy/paste. There are some more advanced techniques you could use to parameterize this so you would only need to set it once, but for a first PR let's keep it simple and just type it in a few times, once per package.

Reply:
Ok - so rather than inherit it from the master / mother module -- we copy it into each of the other 12  sub/add-on modules (and do the same for the other values).  And then maintain all these values manually ?

Reply:
Well I counted 7 other packages... but yes, essentially. I try to keep things as straight forward as possible for first time contributors. There are definitely some nice ways to abstract this out that you'll find around nixpkgs, but unless you really expect `https://redwax.eu/dist/rs/` to change often I don't see it being a problem. And even in that case a tool like `sed` goes a long way.

Feel free to copy/paste from the [code I provided you with.](https://github.com/NixOS/nixpkgs/compare/master...aanderse:httpd-redwax?expand=1)

Reply:
Ok done that. I've left the other (static) values in as inherit from the master. 

Is that ok - or shall I copy those in too ?

Mother:
```
    meta = with stdenv.lib; {
    description = "RedWax CA service module";

    homepage = "https://redwax.eu";
    license = licenses.asl20;
    platforms = platforms.unix;
    maintainers = with maintainers; [dirkx minfrin mvdb];
  };
```
and for the subs:
```
  meta = with stdenv.lib; {
    description = "RedWax CA service module to handle Certificate Signing Requests";

    inherit(mod_ca.meta) license platforms maintainers homepage;
  };

```

Reply:
Gave each module a hardcoded version of the same values - so no longer inheriting.